rdbms-example/vdb-as-maven-artifact/vdb-build/target/vdb.ddl

```diff
@@ -0,0 +1,45 @@
+
+/*
+###########################################
+# START DATABASE customer
+###########################################
+*/
+CREATE DATABASE customer VERSION '1' OPTIONS (ANNOTATION 'Customer VDB');
```

Comment:
Where did VERSION '1' came from btw? Shouldn't this file be ingored?

Reply:
It probably doesn't belong.  @rareddy did you mean to check in the stuff under target?

A longer answer about versions is that 1 is the default version.  For multi-tenet vdb runtimes (embedded, wildfly) the version give you flexibility in how you roll out and connect to deployments of a given vdb.  In the single tenet runtimes it doesn't really have meaning - expect in vdb import scenarios which are still a work in progress.

Reply:
I think that should be removed, it was copied from an example, I missed it. In "vdb-import" scenarios, now the vdb-importer plugin depends on the maven artifact version, not the version defined inside a vdb.